src/main/java/seedu/address/logic/parser/GoCommandParser.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.GoCommand.HISTORY_TAB;
+import static seedu.address.logic.commands.GoCommand.HOME_TAB;
+
+import seedu.address.logic.commands.GoCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new GoCommand object
+ */
+public class GoCommandParser implements Parser<GoCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the GoCommand
+     * and returns a GoCommand object for execution.
+     *
+     * @return the parsed command
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public GoCommand parse(String args) throws ParseException {
+
+        String tabName = args.trim().toLowerCase();
+        if (tabName.equalsIgnoreCase(HOME_TAB)) {
+            return new GoCommand(HOME_TAB);
+        } else if (tabName.equalsIgnoreCase(HISTORY_TAB)) {
+            return new GoCommand(HISTORY_TAB);
+        } else {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, GoCommand.MESSAGE_USAGE));
```

Comment:
since u already use toLowerCase() for the tabName, then i don't think u need to use equalsIgnoreCase, just use equals will do